services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+
+    return MovieSession.objects.create(show_time=movie_show_time,
+                                       movie_id=movie_id,
+                                       cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: datetime = None) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(session_id: int,
+                         show_time: str = None,
```

Comment:
The `show_time` parameter should be a `datetime` object, not a string, to ensure consistency with the `DateTimeField` in the `MovieSession` model. This aligns with the task requirement for 'Data Type Consistency'.